src/components/Map/MapComponent.jsx

```diff
@@ -1,39 +1,72 @@
+import { AppToaster } from '../common/AppToaster';
 
-export const MapComponent = ({ markers }) => {
-    const [, setUserPosition] = useState(null);
-    return (
-        <MapContainer
-            center={mapConfig.initialMapCoordinates}
-            zoom={mapConfig.initialMapZoom}
-            scrollWheelZoom
-            style={{ height: '100%' }}
-            zoomControl={false}
-        >
-            <TileLayer
-                url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-                attribution='&amp;copy <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors'
-                maxZoom={mapConfig.maxMapZoom}
+export const MapComponent = ({ markers, categories, allCheckboxes }) => {
+    const { t } = useTranslation();
+
+    const [userPosition, setUserPosition] = useState(null);
+    const [isListViewOpen, setIsListViewOpen] = useState(false);
+
+    const handleListViewButtonClick = () => {
+        if (!userPosition) {
+            toast.error(t('listViewButtonUserLocation'));
+            return;
+        }
+        setIsListViewOpen(!isListViewOpen);
+    };
+
+    if (isListViewOpen) {
+        return (
+            <AccessibilityTable
+                userPosition={userPosition}
+                setIsAccessibilityTableOpen={setIsListViewOpen}
+                allCheckboxes={allCheckboxes}
             />
-            {window.SHOW_SUGGEST_NEW_POINT_BUTTON && (
-                <Control position="bottomright" prepend>
-                    <SuggestNewPointButton />
-                </Control>
-            )}
-            <MarkerClusterGroup>{markers}</MarkerClusterGroup>
-            <LocationControl setUserPosition={setUserPosition} />
-            <CustomZoomControl position="topright" />
-            {window.SHOW_SEARCH_BAR && <MapAutocomplete />}
-        </MapContainer>
+        );
+    }
+
+    return (
+        <>
+            <AppToaster />
+            <MapContainer
+                center={mapConfig.initialMapCoordinates}
+                zoom={mapConfig.initialMapZoom}
+                scrollWheelZoom
+                style={{ height: '100%' }}
+                zoomControl={false}
+            >
+                <TileLayer
+                    url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
+                    attribution='&amp;copy <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors'
+                    maxZoom={mapConfig.maxMapZoom}
+                />
+                {window.SHOW_SUGGEST_NEW_POINT_BUTTON && (
+                    <Control position="bottomright" prepend>
+                        <SuggestNewPointButton />
+                    </Control>
+                )}
+                <MarkerClusterGroup>{markers}</MarkerClusterGroup>
+                <LocationControl setUserPosition={setUserPosition} />
+                <CustomZoomControl position="topright" />
+                {window.ACCESSIBILITY_TABLE && (
```

Comment:
Make it consistent with others
```suggestion
                {window.SHOW_ACCESSIBILITY_TABLE && (
```